### PR TITLE
Build config templates

### PR DESCRIPTION
**Pull request by itewk**

### CODE REVIEW

File: files/build-config/pipeline-template.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: v1
+kind: Template
+labels:
+  template: pipeline-build-config-tempalte
```
Comment (oybed) [inline files/build-config/pipeline-template.yml]: typo: "tempalte"

File: files/build-config/pipeline-template.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+message: |-
+  Jenkins pipline has been created using: ${GIT_URL}/${JENKINSFILE_PATH}
+metadata:
+  annotations:
+    description: |-
+      Jenkins pipeline build config template.
+  name: pipeline-build-config-tempalte
```
Comment (oybed) [inline files/build-config/pipeline-template.yml]: Same typo here as above

**Review by oybed — COMMENTED** [review]
see inline comments

**Comment by itewk** [comment]
@oybed i fixed the typo and also did some naming convention clean up as well as adding another template we had created in the mean time.

**Review by oybed — APPROVED** [review]
LGTM

**Comment by itewk** [comment]
@oybed thanks for the review and merge

**Pull request merged**